rocAL/rocAL/source/pipeline/master_graph.cpp

```diff
@@ -353,23 +350,16 @@ void MasterGraph::release()
     stop_processing();
     _nodes.clear();
     _root_nodes.clear();
-    _image_map.clear();
+    _tensor_map.clear();
     _ring_buffer.release_gpu_res();
     //shut_down loader:: required for releasing any allocated resourses
-#ifdef ROCAL_VIDEO
-    if(_is_video_loader)
-        _video_loader_module->shut_down();
-    else
-#endif
-        _loader_module->shut_down();
+    _loader_module->shut_down();
     // release all openvx resources.
     vx_status status;
-    for(auto& image: _internal_images)
-        delete image;// It will call the vxReleaseImage internally in the destructor
-    for(auto& image: _output_images)
-        delete image;// It will call the vxReleaseImage internally in the destructor
-    deallocate_output_tensor();
-
+    for(auto& tensor: _internal_tensors)
```

Comment:
We should release the meta-data as well right ?

Reply:
Check if all the other resources that were allocated has been released or not

Reply:
Metadata is not added in the PR